### PR TITLE
fix(locales): use correct lang

### PR DESCRIPTION
**Pull request by posva**

### CODE REVIEW

**Comment by posva** [comment]
Let's wait for the config to be merged on Vue Router next and make sure it works

**Comment by kazupon** [comment]
I same issues vue-i18n-next docs.
I sent PR docsearch-configs (ref: https://github.com/algolia/docsearch-configs/pull/3957#pullrequestreview-635550867)

**Comment by posva** [comment]
Tested with Vue router next, Pinia, and Vite

**Review by kazupon — APPROVED** [review]
~~LGTMです！~~
sorry, my mistake. NVM

File: src/client/theme-default/components/AlgoliaSearchBox.vue [hunk 1]
```diff
@@ -54,11 +58,21 @@ function update(options: any) {
...
   docsearch(
     Object.assign({}, userOptions, {
       container: '#docsearch',
 
-      searchParameters: Object.assign({}, userOptions.searchParameters),
+      searchParameters: Object.assign({}, userOptions.searchParameters, {
+        // pass a custom lang facetFilter to allow multiple language search
+        // https://github.com/algolia/docsearch-configs/pull/3942
+        facetFilters: facetFilters.concat(
+          (userOptions.searchParameters || {}).facetFilters || []
```
Comment (meteorlxy) [inline src/client/theme-default/components/AlgoliaSearchBox.vue:73]: What about `userOptions.searchParameters?.facetFilters || []` ?

Comment (posva) [reply src/client/theme-default/components/AlgoliaSearchBox.vue:73]: I would have used it but the syntax is not transpiled and has very recent support only

Comment (meteorlxy) [reply src/client/theme-default/components/AlgoliaSearchBox.vue:73]: Not transpiled? I thought it would be 🤔

Comment (posva) [reply src/client/theme-default/components/AlgoliaSearchBox.vue:73]: oh, it's actually transpiled when building with vitepress! Not when building vitepress itself. It should be fine then

Comment (posva) [inline src/client/theme-default/components/AlgoliaSearchBox.vue:73]:
```suggestion
          userOptions.searchParameters?.facetFilters || []
```

**Review by kiaking — APPROVED** [review]
👍 👍 👍 👍 👍

**Pull request merged**